pkg/providers/docker.go

```diff
@@ -6,25 +6,25 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
-	"net/url"
 	"strings"
 
+	distreference "github.com/docker/distribution/reference"
 	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/reference"
 	"github.com/moby/moby/client"
 )
 
 const (
-	sh = `docker run --rm -i -t -v ${PWD}:/tmp/cmd -w /tmp/cmd %s/%s:%s "$@"`
+	sh = `docker run --rm -i -t -v ${PWD}:/tmp/cmd -w /tmp/cmd %s:%s "$@"`
```

Comment:
//TODO not sure if this will work on windows. Can we add a TODO here so we don't forget about it?

Reply:
Probably won't. Unless they are on the WSL. I'll add the TODO 👍

Reply:
Makes sense. I can test this and adapt after merging.